uk.co.compucorp.civicrm.hremails/email-templates/src/pages/toil-request.html

```diff
@@ -0,0 +1,62 @@
+---
+layout: leave-absences
+subject: CiviHR TOIL Request
+request-type: TOIL
+---
+
+<div class="request">
+  <row class="collapse">
+    <columns>
+      {{> callout-header title="Leave Request Type"}}
+      <callout class="request-data">
+        {{> request-data key="Status" value="{$leaveStatus}" keySize="4"}}
+        {{> request-data key="Staff Member" value="{contact.display_name}" keySize="4"}}
+
+        {if $leaveRequest->from_date eq $leaveRequest->to_date}
+          {{> request-data key="Date" value="{$fromDate|truncate:10:\'\'|crmDate} {$fromDateType}" keySize="4"}}
+        {else}
+          {{> request-data key="From Date" value="{$fromDate|truncate:10:\'\'|crmDate} {$fromDateType}" keySize="4"}}
+          {{> request-data key="To Date" value="{$toDate|truncate:10:\'\'|crmDate} {$toDateType}" keySize="4"}}
+        {/if}
+
+        {{> request-data
+          key="No. TOIL Days Requested"
+          value="{$leaveRequest->toil_to_accrue} {if $leaveRequest->toil_to_accrue > 1}days{else}day{/if}"
+          class="request-data-toil"
+          keySize="4"
+          keyTag="div"
+        }}
+      </callout>
+      <spacer></spacer>
+      <button href="{$leaveRequestLink}" class="expanded alert">View This Request</button>
+      {if $leaveComments}
+        <hr>
+        {{> callout-header title="Request comments"}}
+        <callout class="callout-no-padding">
+          <div class="request-comments">
+            {foreach from=$leaveComments item=value key=label}
+              {{> request-comment commenter="{$value.commenter}" createdAt="{$value.created_at|crmDate}" text="{$value.text}"}}
+            {/foreach}
+          </div>
+        </callout>
+      {/if}
+      {if $leaveFiles}
+        <hr>
+        {{> callout-header title="Other files recorded in this request"}}
+        <callout>
+          <div class="request-attachments">
+            {foreach from=$leaveFiles item=value key=label}
+              {{> request-attachment name="{$value.name}" uploadDate="{$value.created_at|crmDate}"}}
+            {/foreach}
+          </div>
+        </callout>
+      {/if}
+    </columns>
+  </row>
+  <row class="collapse">
```

Comment:
This code is same in leave request tamplate. Do we have option to import partials? then move the common code there.